doc/api/repl.md

```diff
@@ -411,6 +411,8 @@ added: v0.1.91
     `SIGINT` is received, i.e. `Ctrl+C` is pressed. This cannot be used together
     with a custom `eval` function. Defaults to `false`.
 
+If `options` is a string, then it specifies the input prompt.
```

Comment:
I've tried to mirror others argument variant notes, for example in `fs.md`:

https://github.com/nodejs/node/blame/master/doc/api/fs.md#L473
https://github.com/nodejs/node/blame/master/doc/api/fs.md#L629
https://github.com/nodejs/node/blame/master/doc/api/fs.md#L676
https://github.com/nodejs/node/blame/master/doc/api/fs.md#L1364
https://github.com/nodejs/node/blame/master/doc/api/fs.md#L1916

Would the more verbose note be a duplicate of this explanation above:

https://github.com/nodejs/node/blame/master/doc/api/repl.md#L375

Maybe, you could suggest the explanation? As for example, maybe I could add some.